docs: update readme for UTRP v2 prod release (#241)
Longhorn-Developers/UT-Registration-Plus: README.md

- Semantic-Release
- Custom Messaging & Storage Wrappers

## Contributing

Contributions are welcome and encouraged! To contribute, please follow these steps:

1. Fork the repository
2. Create a new branch (`git checkout -b feature/your-feature`)

> 3. Make your changes and ensure the code follows the linting and formatting rules (`bun run lint` and `bun run format`)

Author: ```suggestion
3. Make your changes and ensure the code follows the linting and formatting rules (`pnpm run lint` and `pnpm run prettier`)
```

<type>(<scope>): <subject>
```

Where:

- **type**: One of the following:
  - `feat`: A new feature
  - `fix`: A bug fix
  - `docs`: Documentation changes
  - `style`: Code formatting (white-space, formatting, missing semi-colons, etc.)
  - `refactor`: Code restructuring (neither fixes a bug nor adds a feature)
  - `test`: Adding or modifying tests
  - `chore`: Changes to the build process or auxiliary tools

Example:

```
feat(auth): add login functionality
fix(ui): correct button alignment
```

### Branch Naming

Branch names should follow the format:

```
<type>/<short-description>
```

Examples:

- `feat/user-login`
- `fix/navbar-layout`
- `docs/update-readme`

## License

This project is licensed under the MIT License. See the [LICENSE](./LICENSE.md) file for more details.

## Code of Conduct [TODO]

We adhere to a strict code of conduct. By participating, you are expected to uphold this code. Please read [CODE_OF_CONDUCT.md](./CODE_OF_CONDUCT.md) for details.

## Acknowledgements

This project leverages several powerful open-source tools and libraries. Special thanks to the developers behind:
- React 18
- TypeScript
- Vite 5
- ESLint
- Prettier
- Storybook
- Semantic-Release

> - [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner)
> - [Longhorn Developers](https://github.com/Longhorn-Developers)

Author: ```suggestion
- UT Registration Plus v1 with @sghsri as the original author for creating the foundation of UTRP and impacting thousands of UT students positively via his Chrome Extension
- [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner) was archived and it's developer @doprz joined UT Registration Plus in 2023
- [Longhorn Developers](https://github.com/Longhorn-Developers) organization was created in 2024 for further development of UTRP and other student-led projects.
- All UTRP devs, beta testers, and anyone who helped support this project
```

> [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner)'s codebase and development was adquired by UT Registration Plus in 2023 in order to help serve The University of Texas at Austin's community more effectively in addition to it leading to the creation of the [Longhorn Developers](https://github.com/Longhorn-Developers) organization for further development of UTRP and other student-led projects.

Author: Hmm, I think if I remove the word "codebase" and just keep "development" it would better reflect it. Thoughts?

> [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner)'s codebase and development was adquired by UT Registration Plus in 2023 in order to help serve The University of Texas at Austin's community more effectively in addition to it leading to the creation of the [Longhorn Developers](https://github.com/Longhorn-Developers) organization for further development of UTRP and other student-led projects.

Reviewer: No it sounds like UTRP 2.0 was dependent on Planner.

> [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner)'s codebase and development was adquired by UT Registration Plus in 2023 in order to help serve The University of Texas at Austin's community more effectively in addition to it leading to the creation of the [Longhorn Developers](https://github.com/Longhorn-Developers) organization for further development of UTRP and other student-led projects.

Author: ```suggestion
[UT Registration Planner](https://github.com/doprz/UT-Registration-Planner)'s was archived and it's developer joined UT Registration Plus in 2023
- [Longhorn Developers](https://github.com/Longhorn-Developers) organization was created in 2024 for further development of UTRP and other student-led projects.
```

```

Where:

- **type**: One of the following:
  - `feat`: A new feature
  - `fix`: A bug fix
  - `docs`: Documentation changes
  - `style`: Code formatting (white-space, formatting, missing semi-colons, etc.)
  - `refactor`: Code restructuring (neither fixes a bug nor adds a feature)
  - `test`: Adding or modifying tests
  - `chore`: Changes to the build process or auxiliary tools

Example:

```
feat(auth): add login functionality
fix(ui): correct button alignment
```

### Branch Naming

Branch names should follow the format:

```
<type>/<short-description>
```

Examples:

- `feat/user-login`
- `fix/navbar-layout`
- `docs/update-readme`

## License

This project is licensed under the MIT License. See the [LICENSE](./LICENSE.md) file for more details.

## Code of Conduct [TODO]

We adhere to a strict code of conduct. By participating, you are expected to uphold this code. Please read [CODE_OF_CONDUCT.md](./CODE_OF_CONDUCT.md) for details.

## Acknowledgements

This project leverages several powerful open-source tools and libraries. Special thanks to the developers behind:
- React 18
- TypeScript
- Vite 5
- ESLint
- Prettier
- Storybook
- Semantic-Release

> [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner)'s was archived and it's developer joined UT Registration Plus in 2023

Author: ```suggestion
- [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner)'s was archived and it's developer joined UT Registration Plus in 2023
```

> - [Longhorn Developers](https://github.com/Longhorn-Developers) organization was created in 2024 for further development of UTRP and other student-led projects.

Author: ```suggestion
- UT Registration Plus v1 with @sghsri as the original author for creating the foundation of UTRP and impacting thousands of UT students positively via his Chrome Extension
- [UT Registration Planner](https://github.com/doprz/UT-Registration-Planner) was archived and it's developer @doprz joined UT Registration Plus in 2023
- [Longhorn Developers](https://github.com/Longhorn-Developers) organization was created in 2024 for further development of UTRP and other student-led projects.
- All UTRP devs, beta testers, and anyone who helped support this project
```
